Resources/views/environment/index.html.twig

```diff
@@ -75,7 +75,7 @@
 							<th>Action</th>
 						</tr>
 						{% for environment in environments %}
-							{{ 'Environmen\'s info'|debug({name: environment.name}) }}
+							{{ "Environmen's info"|debug({name: environment.name}) }}
```

Comment:
This is an interesting update. I must be from the past: https://stackoverflow.com/questions/3446216/what-is-the-difference-between-single-quoted-and-double-quoted-strings-in-php